content/community-update/community-update/dcp-updates.md

```diff
@@ -6,6 +6,23 @@ title: DCP Updates
 
 # DCP Platform Updates
 
+#### October 19, 2022
+
+### Release Highlights
+* New COVID-19 data is available in the project ["COVID-19 tissue atlases reveal SARS-CoV-2 pathology and cellular targets."](https://data.humancellatlas.org/explore/projects/61515820-5bb8-45d0-8d12-f0850222ecf0)
+
+* The new project ["Genome-Wide DNA Hypermethylation in the Wound-Edge of Chronic Wound Patients Opposes Closure by Impairing Epithelial to Mesenchymal Transition"](https://data.humancellatlas.org/explore/projects/34ec845b-cd7a-4c43-99e4-d2932d5d85bb) combines spatial transcriptomic data with methylome data. 
```

Comment:
It looks like we have been inconsistent about using quotes around project names in the Release Highlights. It's not a big deal but we might want to revisit this for consistency.

Reply:
Good point and thanks! Removed quotes for now around project titles.